packages/node_modules/@node-red/editor-client/src/js/ui/editor.js

```diff
@@ -953,8 +953,70 @@ RED.editor = (function() {
 
         var i,row;
 
+        if (node.type === "subflow") {
+            var categoryRow = $("<div/>", {
+                class: "form-row"
+            }).appendTo(dialogForm);
+            $("<i/>", {
+                class: "fa fa-folder-o"
+            }).appendTo(categoryRow);
+            $("<span/>").text(" ").appendTo(categoryRow);
+            $("<label/>", {
+                for: "subflow-appearance-input-category",
+                "data-i18n": "editor:subflow.category"
+            }).appendTo(categoryRow);
+            var categorySelector = $("<select/>", {
+                id: "subflow-appearance-input-category"
+            }).css({
+                width: "250px"
+            }).appendTo(categoryRow);
+            $("<input/>", {
+                type: "text",
+                id: "subflow-appearance-input-custom-category"
+            }).css({
+                display: "none",
+                "margin-left": "10px",
+                width: "calc(100% - 250px)"
+            }).appendTo(categoryRow);
+
+            var categories = RED.palette.getCategories();
+            categories.sort(function(A,B) {
+                return A.label.localeCompare(B.label);
+            })
+            categories.forEach(function(cat) {
+                categorySelector.append($("<option/>").val(cat.id).text(cat.label));
+            })
+            categorySelector.append($("<option/>").attr('disabled',true).text("---"));
+            categorySelector.append($("<option/>").val("_custom_").text(RED._("palette.addCategory")));
+
+            $("#subflow-appearance-input-category").on("change", function() {
+                var val = $(this).val();
+                if (val === "_custom_") {
+                    $("#subflow-appearance-input-category").width(120);
+                    $("#subflow-appearance-input-custom-category").show();
+                } else {
+                    $("#subflow-appearance-input-category").width(250);
+                    $("#subflow-appearance-input-custom-category").hide();
+                }
+            })
+
+            $("#subflow-appearance-input-category").val(node.category||"subflows");
+            var userCount = 0;
+            var subflowType = "subflow:"+node.id;
+
+            RED.nodes.eachNode(function(n) {
+                if (n.type === subflowType) {
+                    userCount++;
+                }
+            });
+            $("#red-ui-editor-subflow-user-count")
+                .text(RED._("subflow.subflowInstances", {count:userCount})).show();
+        }        
+        
         $('<div class="form-row">'+
+            '<i class="fa fa-tag"> '+
```

Comment:
This tag doesn't have a closing `</i>` tag which breaks the appearance of this row:

<img width="106" alt="Local_Node-RED" src="https://user-images.githubusercontent.com/51083/61122602-238ef100-a49a-11e9-896d-4a58eccb8647.png">


In fact, we don't have icons on this tab at all... please don't add them.